src/ripple/app/tx/impl/AMMDeposit.cpp

```diff
@@ -0,0 +1,682 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012-2022 Ripple Labs Inc.
```

Comment:
Should probably only have 2022 as the year

Reply:
I don't think this matters. But FWIW, I would actually suggest `2012-2023`.

Reply:
Not to bikeshed, but... I looked at the lines that have the `year, year` format, and the overwhelming majority of them are old: `2012, 2013`. With about 30 exceptions, nothing after 2013 uses this format.

```
$ grep -rh "Copyright (c).*,.*Ripple Labs" src/ripple/ | sort | uniq -c
    510     Copyright (c) 2012, 2013 Ripple Labs Inc.
      1     Copyright (c) 2012, 2013, 2019 Ripple Labs Inc.
      4     Copyright (c) 2012, 2015 Ripple Labs Inc.
      6     Copyright (c) 2012, 2017 Ripple Labs Inc.
      3     Copyright (c) 2012, 2018 Ripple Labs Inc.
      2     Copyright (c) 2012, 2019 Ripple Labs Inc.
     12     Copyright (c) 2012, 2020 Ripple Labs Inc.
      1     Copyright (c) 2012, 2023 Ripple Labs Inc.
```

Everything after that has the copyright year that the file was originally created, either by itself or as the first of a range with the year of the last update. (e.g. `2022` or `2012-2022`)

```
$ grep -rh "Copyright (c).*Ripple Labs" src/ripple/ | sort | uniq -c
```

In the time that I've been here, any new files that I've seen had their copyright year as just the year of creation.

Reply:
Ok, I'll update all new AMM files with just the year of creation.